src/sentry/issues/grouptype.py

```diff
@@ -332,6 +345,7 @@ class PerformanceDurationRegressionGroupType(PerformanceGroupTypeDefaults, Group
     description = "Transaction Duration Regression (Experimental)"
     noise_config = NoiseConfig(ignore_limit=0)
     category = GroupCategory.PERFORMANCE.value
+    enable_auto_resolve = False
```

Comment:
Here and below, we can skip setting to False explicitly since the default for `enable_auto_resolve` is False.

Reply:
I put it here because it inherits from PerformanceGroupTypeDefaults which sets enable_auto_resolve to true. This way we don't have to explicitly set all the legacy performance group types to enable_auto_resolve = True.

Reply:
gotcha, in that case let’s do one of the following:
1. Set `enable_auto_resolve = True` explicitly for EACH legacy performance type so it’s very clear
2. Add a comment here and below to explain why we’re re-setting the value to the default.

I prefer #1 since it leaves no room for confusion, but either approach works!

Reply:
From Snigdha:

> gotcha, in that case let’s do one of the following:
> 1. Set `enable_auto_resolve = True` explicitly for EACH legacy performance type so it’s very clear
> 2. Add a comment here and below to explain why we’re re-setting the value to the default.

> I prefer 1 since it’s very clear, but either approach works!